packages/react-native/React/CoreModules/RCTAlertController.mm

```diff
@@ -43,9 +43,13 @@ - (UIWindow *)alertWindow
 
 - (void)show:(BOOL)animated completion:(void (^)(void))completion
 {
-  UIUserInterfaceStyle style = RCTSharedApplication().delegate.window.overrideUserInterfaceStyle
-      ? RCTSharedApplication().delegate.window.overrideUserInterfaceStyle
-      : UIUserInterfaceStyleUnspecified;
+  UIUserInterfaceStyle style = self.overrideUserInterfaceStyle;
+  if (style == UIUserInterfaceStyleUnspecified) {
+    UIUserInterfaceStyle style = RCTSharedApplication().delegate.window.overrideUserInterfaceStyle
+        ? RCTSharedApplication().delegate.window.overrideUserInterfaceStyle
+        : UIUserInterfaceStyleUnspecified;
+  }
```

Comment:
Internally, our linter is much more stricter than externally.
You are redeclaring the exact same `UIUserInterfaceStyle style` variable twice and our internal builds are failing. Could you please aplly the following suggestion? 🙏 
```suggestion
  UIUserInterfaceStyle style = self.overrideUserInterfaceStyle;
  if (style == UIUserInterfaceStyleUnspecified) {
    style = RCTSharedApplication().delegate.window.overrideUserInterfaceStyle
        ? RCTSharedApplication().delegate.window.overrideUserInterfaceStyle
        : UIUserInterfaceStyleUnspecified;
  }
```

Reply:
@cipolleschi Done.